### DISCUSSION TITLE
插件相关问题

### DISCUSSION THREAD

**Discussion by 204065248** [wangeditor-team/wangEditor#5073] [Q&A] [answered]
请问这段代码是什么意思，没太读懂
![image](https://user-images.githubusercontent.com/17307843/201486770-eaa23358-bbde-474c-8f16-3dece334969d.png)

**Comment by wangfupeng1988** [comment, marked as answer]
首先，你得明白什么是 void elem

**Reply by 204065248** [reply]
这个我看明白了

**Comment by wangfupeng1988** [comment]
看明白 void elem 就简单了。这段代码的意思是：当选中 void elem 时，不去重写 insertText ，直接执行原有的 insertText 即可。

**Reply by 204065248** [reply]
嗯嗯，明白。👍
